.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "scp"]
+	path = scp
+	url = git@github.com:StreamSpace/scp.git
+    branch = lcBump.0
```

Comment:
This should be lightClient?

Reply:
yes. once scp gets merged will change this back.